releases/v3.0.0.toml

```
@@ -0,0 +1,58 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v3.0.0-rc.4"
+
+pre_release = true
```

Does this need to be set to false?

```suggestion
pre_release = false
```